swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        return False
+
+    def get_by_category(self, category):
+        result = [item for item in self.inventory if item.category is category]
+        return result
```

Comment:
Great to see you're practicing list comprehensions! Very succinct way to handle building up the list 🙂

Since we don't perform any other operations with `result`, we could also directly return the result of the comprehension:
```suggestion
        return [item for item in self.inventory if item.category == category]
```